src/ui/services/networkService.ts

```diff
@@ -174,6 +184,21 @@ export class NetworkService extends EventEmitter {
                 d.setUTCMilliseconds(data.timestamp);
                 this.emit("chat", d, data.sender, data.message); 
             });
+            this.currentLobbyConnection.on("start-game", function(GameServerPort){
```

Comment:
Don't write inline functions here. Extract this to it's own function, like most of the other handlers have done.